src/App.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="app">
+    <app-header></app-header>
+    <router-view></router-view>
+  </div>
+</template>
+<script>
+import appHeader from './components/header.vue';
+export default {
+  components: {
+    'app-header': appHeader,
+  },
```

Comment:
Since there is only one key in the object, it can be written as:
`components: { 'app-header': appHeader },`

When you have multiple keys in an object you can write in a new line.